schema.sql

```diff
@@ -8,5 +8,6 @@ name VARCHAR(100) NOT NULL,
 date_of_birth DATE NOT NULL,
 escape_attempts INT NOT NULL,
 neutered bool NOT NULL,
-weight_kg decimal NOT NULL
+weight_kg decimal NOT NULL,
+species VARCHAR(255);
```

Comment:
- [ Required ] I think it would be nice if you change the structure of this table using the `ALTER TABLE` command, if this was a database in production it would not be advisable to add the `species` column directly like this. Please use the `ALTER TABLE` command as shown in the code sample below, this is just a suggestion please feel free to use another method.

## Example code
```SQL
ALTER TABLE animals ADD species varchar(255);
```